libs/service-portal/finance/src/components/DocumentScreen/DocumentScreen.tsx

```diff
@@ -54,13 +56,16 @@ const DocumentScreen: FC<Props> = ({ title, intro, listPath }) => {
   const [page, setPage] = useState(1)
   const [fromDate, setFromDate] = useState<string>()
   const [toDate, setToDate] = useState<string>()
+  const [q, setQ] = useState<string>('')
 
   const [loadDocumentsList, { data, loading, called, error }] = useLazyQuery(
     getFinanceDocumentsListQuery,
   )
 
   const billsDataArray: DocumentsListItemTypes[] =
-    data?.getDocumentsList?.documentsList || []
+    (data?.getDocumentsList?.documentsList &&
+      billsFilter(data?.getDocumentsList?.documentsList, q)) ||
```

Comment:
```suggestion
      billsFilter(data.getDocumentsList.documentsList, q)) ||
```

These ? checks are redundant, you just establised that documentList is truey in the line above

Generally: "?" checks may be cheap, but they're not completely free. They make the bundle bulkier, and add tiny bit of extra work.

BTW, this is something I see done many places. Not aimed at you specifically Þórður. :-)

Reply:
Yes Totally! I'll fix it.